<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <widget class="QTabWidget" name="authorization_tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="loginTab">
    <attribute name="title">
     <string>Вход</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="loginNickname_lineEdit">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Никнейм</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loginPassword_lineEdit">
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="save_checkBox">
        <property name="text">
         <string>Запомнить мои данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="login_pushButton">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="signupTab">
    <attribute name="title">
     <string>Регистрация</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>241</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="signupNickname_lineEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Никнейм</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="signupName_lineEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Имя</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sex_comboBox">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="placeholderText">
         <string>Укажите ваш пол</string>
        </property>
        <item>
         <property name="text">
          <string>М</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ж</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="signupPassword_lineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="signupPasswordRepeat_lineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Повторите пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signup_pushButton">
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="serverAddr_lineEdit">
      <property name="placeholderText">
       <string>Адрес сервера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="serverPort_lineEdit">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Порт сервера</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="connectToServer_pushButton">
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetSettings_pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сбросить сохраненные настройки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/ResetIcon.png</normaloff>:/ResetIcon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
